ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-fcos-master.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
+  base:
+    name: "4.3"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: machine-config-operator
+promotion:
+  name: "4.3"
+  namespace: fcos
```

Comment:
this is probably fine as a promotion target.  I think we'll want a separate job that takes over from the okd promotion flow that combines the three custom images here (machine-os-content, machine-config-operator, and installer) tests them with the latest okd images, and promotes them.

Once we have these publishing to fcos I will update the release controller so that it stops publishing the three images from ocp to okd, and instead we'll side load them via the job I mentioned.